retrieve_files.sh

```diff
@@ -157,10 +157,25 @@ log "INFO" "File unzip process complete"
 
 log "INFO" "Starting file parsing process"
 
-#for f in "$dropLocation/*"
-#do
-  #  curl -x PUT --data-binary @$dropLocation/$filename http://192.168.99.100:9998/tika --header "Content-type: application/pdf" > ${filename%.*}.txt
-#done       
+for f in $dropLocation/*
+do
+  if [[ $f == *.zip ]]
```

Comment:
You could eliminate this check here by making your loop include the extension:

`for f in $dropLocation/*.pdf`